diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -137,3 +137,64 @@ int DifferentialFunctionComparator::cmpAttrs(const AttributeList L,
     }
     return FunctionComparator::cmpAttrs(LNew, RNew);
 }
+
+/// Handle comparing of memory allocation function in cases where the size
+/// of the composite type is different
+int DifferentialFunctionComparator::cmpOperations(
+    const Instruction *L, const Instruction *R, bool &needToCmpOperands) const {
+    int Result = FunctionComparator::cmpOperations(L, R, needToCmpOperands);
```

Comment:
Wouldn't it make sense to run the original cmpOperations at the end of this function? Now, it is run every time, even when its result is not used.